app/api/resources/mentorship_relation.py

```diff
@@ -92,12 +93,12 @@ def post(cls):
         is_valid = SendRequest.is_valid_data(data)
 
         if is_valid != {}:
-            return is_valid, HTTPStatus.BAD_REQUEST
+            return is_valid, HTTPStatus.BAD_REQUEST.value
 
         response = DAO.create_mentorship_relation(user_sender_id, data)
 
         # if the mentorship relation creation failed dont send email and return
-        if response[1] != HTTPStatus.CREATED:
+        if response[1] != HTTPStatus.CREATED.value:
```

Comment:
shall I remove .value from here too.
@devkapilbansal

Reply:
No, it looks fine

Reply:
Done @devkapilbansal